netmaster/objApi/apiController.go

```diff
@@ -737,38 +737,40 @@ func (ac *APIController) TenantCreate(tenant *contivModel.Tenant) error {
 		return err
 	}
 
-	// Create private network for the tenant
-	err = contivModel.CreateNetwork(&contivModel.Network{
-		Key:         tenant.TenantName + ":" + "private",
-		IsPublic:    false,
-		IsPrivate:   true,
-		Encap:       "vxlan",
-		PktTag:      1001,
-		Subnet:      "10.1.0.0/16",
-		Gateway:     "10.1.254.254",
-		NetworkName: "private",
-		TenantName:  tenant.TenantName,
-	})
-	if err != nil {
-		log.Errorf("Error creating privateNet for tenant: %+v. Err: %v", tenant, err)
-		return err
-	}
+	if tenant.TenantName == "default" {
```

Comment:
just for my understanding, what is the reason to create a private network just for 'default' tenant? Should we follow the same for public network as well below?